daemons/lvmdbusd/utils.py

```diff
@@ -534,20 +534,26 @@ def add_no_notify(cmdline):
 # ensure all dbus library interaction is done from the same thread!
 
 
-def _async_result(call_back, results):
-	log_debug('Results = %s' % str(results))
-	call_back(results)
+def _async_handler(call_back, parameters):
+	params_str = ", ".join([str(x) for x in parameters])
```

Comment:
The square brackets can be dropped here.

Reply:
And probably also the ``str(x)`` call, ``join()`` should do this on it's own, AFAICT.

Reply:
I'll see what I can simplify here

Reply:
str(x) appears to be required, for example if I pass a function itself as a parameter, join blows up on that.